index.html

```diff
@@ -5,12 +5,14 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-    <title>Document</title>
-
-    <section></section>
-    </head>
-    <section><img src="photos/photoNicole.jpeg" alt="Nicole" width="300" height="300" style="border-radius: 50%"></section>
-    <section>
+    <title>Nicole's Profile Assignment</title>
+</head>
+   
+<body>
+    <div id="mainSection" style="padding:10%; border: 5px solid magenta;">
+    <section><img src="photos/photoNicole.jpeg" alt="Nicole" width="300" height="300" style="border-radius: 50%; position:relative; left:40%; margin-top:3%;"></section>
+   
+        <section>
         <h1 style="color:purple;">ABOUT:</h1>
         <h2>Hi, my name is Nicole Bradley. I love computer coding. It is my dream to one day be a game developer. 
```

Comment:
If you use any of the `h` tags, the contents should be short. This should probably be a `p` or `section` or `div`.